homeassistant/components/doorbird/__init__.py

```diff
@@ -51,72 +56,24 @@
 )
 
 
-def setup(hass, config):
+async def async_setup(hass: HomeAssistant, config: dict):
     """Set up the DoorBird component."""
 
+    hass.data.setdefault(DOMAIN, {})
+
     # Provide an endpoint for the doorstations to call to trigger events
     hass.http.register_view(DoorBirdRequestView)
```

Comment:
Future PR we should migrate this to webhooks.

Reply:
Already on the list after rachio (needs some surgery to clean it up a bit first)